### ISSUE TITLE
Support domain + Negation filters

### ISSUE THREAD

**Issue by cedricbraekevelt** [code4me/4me-sdk-csharp#51]
Dear Klaas,

First of all thank you very much for the development on this API, it's a great help.
I'm using the latest nuget package of your API at this moment.
I want to ask 2 questions:
1) I'm having issues with the negation filter. It requires 2 values (throws "Unable to get open tickets in info for 4Me The Negation filter requires a two values." error).  I'm using this in the following situation:
`
FilterCollection filters = new FilterCollection();
filters.Add(new Filter("Status", FilterCondition.Negation, RequestStatus.Completed)); 
`
However I have no idea which extra attribute I should add, I've noticed in the source code it requires 2 attributes to proceed. This however makes it impossible to retrieve all our open tickets. 
2) I've noticed the support domain for a ticket has been changed to a readonly value. However we used to assign the supportdomain while the tickets were being generated. Is this a bug or is it meant to be? I saw in the source code you noted: 
"Used to specify the support domain account ID in which the request is to be registered. This parameter needs to be specified when the current user’s Person record is registered in a directory account. The ID of a 4me account can be found in the ‘Account Overview’ section of the Settings console."
However we surely used to be able to create requests with other supportdomain's then the one assigned to the user. 

Kind regards,
Cédric

**Comment by klaasvandeweerdt** [comment]
Dear Cédric,

Thanks for the feedback.

1. This seems to be a bug in my code. Negation only requires one value. The only filter that requires 2 arguments would be ```GreaterThanOrEqualToAndLessThanOrEqualTo```. I adapted the code and ran some tests, instead of an exception it now queries the endpoint ```GET "https://api.4me.qa/v1/requests?per_page=100&sort=updated_at&status=!completed"``` and returns the results.
2. Looks like a bug as well. As you mentioned and according to the documentation, this property should not be read only. I adapted the code an made it read/write.

Local commits are done for both issues. I will push the changes as soon as possible. The NuGet (2.0.2) will be updated over the weekend, this way I can include (possible) changes 4me may publish in their weekly release.

Kind regards

Klaas
